dub.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "sdc",
+    "description": "The snazzy D compiler. Currently only sdfmt (sdc:sdfmt) is available via dub",
+	"targetType": "none",
+	"subPackages": [
+		{
+			"name": "sdfmt",
+			"targetType": "executable",
+            "mainSourceFile": "src/driver/sdfmt.d",
+            "sourcePaths": ["src/format/", "src/source", "src/config"]

```

Comment:
Just src, no?

Reply:
By default it looks in `src` anyway, but if you do that it tries to compile every single file in one go for some reason.

Reply:
Nice, so what was missing for D was a package manager tied to a build system that don't really know how to handle dependencies.

Reply:
No, it is because you mixed tabs ans spaces.

Just run prettier --use-tabs --tab-width 4 dub.json

Reply:
> Nice, so what was missing for D was a package manager tied to a build system that don't really know how to handle dependencies.

yes, apparently. Oh and one that can't do parallel builds with more than 2 threads most of the time